www/widgets/rt.js

```diff
@@ -164,55 +166,90 @@ class ssl {
 }
 
 class tkinter { 
-    static Tk = wrap_class(class {
-        constructor() {
-            this.elt = rt_constants.ROOT_CANVAS;
+    static Tk = function(...args) {
+        if (rt_constants.WINDOW) {
+            console.log('reused')
```

Comment:
Do we need to keep this debug print? (If we do, we should make it more informative at least.)